web/pandas/versions.json

```
@@ -1,30 +1,37 @@
 [
     {
         "name": "dev",
-        "version": "docs/dev"
+        "version": "dev",
+        "url": "docs/dev/"
```

Not sure how this is used, but if it's just added to the `href` attribite of the links, I think we want to have a heading slash in it.

```suggestion
        "url": "/docs/dev/"
```
In html, a link that doesn't start with a slash is relative. So, if the user is in `/docs/user-guide/` the address resolved with the url in your changes would be `/docs/user-guide/docs/dev/`. Starting it with the slash will resolve it to that absolute url for the current domain.

---

Thx :)